examples/column_view_datagrid/grid_cell/grid_cell.ui

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="GridCell" parent="GtkBox">
+    <property name="orientation">horizontal</property>
+    <property name="spacing">12</property>
+    
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <property name="margin-top">6</property>
+        <property name="margin-bottom">6</property>
+        <child>
+          <object class="GtkLabel" id="name">
```

Comment:
Not sure if this example should also target GTK version 4.6 and below, but with GTK 4.8 there's `GtkInscription` which is a lot more appropriate in this case to ensure good performance with this widget.

Reply:
I think it is fine to use gtk4.8 here, I am planning to do a release of gtk4-rs soonish with the new APIs that will land in 4.8

Reply:
I just went ahead and removed the margin and such now

Reply:
@cmdcolin Maybe there was a misunderstanding here. I meant to use the new `GtkInscription` widget instead of the current `GtkLabel`. Inscription is a new widget in GTK 4.8 (not yet released) which is pretty similar to a label, but it's a lot simpler internally and it's specifically designed to be used in this kind of cells to allow a smooth scrolling when there are lots of items (like in this case).

Reply:
>  GTK 4.8 (not yet released)

Already released by the way

Reply:
> > GTK 4.8 (not yet released)
> 
> Already released by the way

Oh, I missed that :)

Reply:
I see, I get an error trying to use gtk::Inscription that it's not found but definitely open to it

```
   Compiling gtk4-rs-examples v0.1.0 (/home/cdiesh/src/gtk4-rs/examples)
error[E0412]: cannot find type `Inscription` in crate `gtk`
  --> examples/column_view_datagrid/grid_cell/imp.rs:12:34
   |
12 |     pub name: TemplateChild<gtk::Inscription>,
   |                                  ^^^^^^^^^^^ not found in `gtk`

error[E0283]: type annotations needed
  --> examples/column_view_datagrid/grid_cell/imp.rs:12:5
   |
8  | #[derive(Debug, Default, CompositeTemplate)]
   |                 ------- in this derive macro expansion
...
12 |     pub name: TemplateChild<gtk::Inscription>,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type
   |
   = note: cannot satisfy `_: std::default::Default`
   = note: this error originates in the derive macro `Default` (in Nightly builds, run with -Z macro-backtrace for more info)

Some errors have detailed explanations: E0283, E0412.
For more information about an error, try `rustc --explain E0283`.
error: could not compile `gtk4-rs-examples` due to 2 previous errors

```